example/src/api/index.ts

```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
+import { events } from './events';
+import { jiterWebhookEvent } from './webhooks';
+
+export const api = Router();
+
```

Comment:
Can you add a simple api.get('/') route that returns all the available endpoints so if they hit `http://localhost:8000` they don't get an error

Reply:
```typescript
app.get('/', (req, res) => res.json({ message: 'Welcome to the Jiter & Express TS example, checkout the /events and /webhooks endpoints!' }));
```